query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/ExecutionContextTest.java

```diff
@@ -156,12 +160,10 @@ public void testReferencedColumns() {
             .setOperator(Operator.EQ);
     Filter startTimeFilter =
         QueryRequestBuilderUtils.createTimeFilter(
-            "Trace.start_time_millis",
-            Operator.GT,
-            System.currentTimeMillis() - 1000 * 60 * 60 * 24);
+            "Trace.start_time_millis", Operator.GT, currentTimeInMillis - 1000 * 60 * 60 * 24);
```

Comment:
no math! make the quantities human readable literals. Such as `"Trace.start_time_millis", Operator.GT, MILLISECONDS.convert(1, HOURS)` then below, `"Trace.end_time_millis", Operator.LT, MILLISECONDS.convert(2, HOURS))`